parser.js

```diff
@@ -719,6 +729,8 @@
       //      CharacterClass
       //      ( GroupSpecifier Disjunction )
       //      ( ? : Disjunction )
+      //      ( ? RegularExpressionFlags : Disjunction )
```

Comment:
Nit: Technically `( ? : Disjunction )` is included in `( ? RegularExpressionFlags : Disjunction )`, maybe we can remove the former production.